_posts/2022-11-17-Multiple-Authentication.md

```diff
@@ -0,0 +1,53 @@
+
+* Simultaneous login with basic authentication (username and password) and SSO
+* An integrated login/logout experience regardless of your authentication methodology
+* Flexibility for Dashboards security admins to customize the login page options
+* Support for identity redundancy across multiple IdPs to ensure high availability
+* Simplified troubleshooting of external IdP-related issues
+
+## Use cases
+
+Customers using dedicated third-party enterprise-based IdPs to manage Dashboards user identities can now use OpenSearch as an internal IdP for admin, security, and privileged accounts and a single enterprise-based external IdP to maintain authentication information for general Dashboards user accounts. Larger enterprises that have redundancy needs can enable multiple enterprise-based IdPs, for example, an enterprise OIDC IdP and an enterprise SAML IdP.
+
+Additionally, you can now add multiple external IdPs to meet different needs. For example, OpenSearch can serve as an internal IdP to maintain authentication information for admin and security accounts, an enterprise-based IdP for privileged and analyst accounts, and a social-based IdP for limited read-only accounts that can only view specific data.
+
+If you have any additional use cases you think this feature can meet, we would love to hear about them! Please visit the [OpenSearch Forum](https://forum.opensearch.org/t/feedback-multiple-authentication-options-for-opensearch-dashboards/11508) to leave feedbacks for us.
+
+## How can you use this feature?
+
+OpenSearch Dashboards admins can enable single or multiple authentication options on demand and customize the integrated login UI by editing opensearch_dashboards.yml. For the essential settings required to set up multiple authentication, see [**Multiple Authentication Options**](https://opensearch.org/docs/latest/security-plugin/configuration/multi-auth/). Once these options are configured, the login UI will be automatically updated. For example, the login UI below shows both basic authentication and SSO enabled.
+
+![Default Login UI](/assets/media/blog-images/2022-11-14-multiple-authentication/default-login-ui.png){: .img-fluid}
+
+## How does this feature work?
+
+When Dashboards bootstraps and loads security settings from `opensearch-dashboards.yml`, the Dashboards client can detect whether Single-Authentication Mode or Multiple-Authentication Mode is enabled by evaluating the Multiple Authentication feature flag and authentication type setting:
+
+* If **Single-Authentication Mode** is detected, Dashboards registers a single authentication handler with the client based on the authentication type defined by **opensearch_security.auth.type** setting. When Dashboards users log in, all requests flow to the dedicated IdP to complete identification and authentication.
```

Comment:
This should be "... defined by the **opensearch_security.auth.type** setting."
(add the word "the".)

Reply:
My bad. Fix it. Thanks!